helm/dagger/values.yaml

```diff
@@ -0,0 +1,50 @@
+nameOverride: ""
+fullnameOverride: ""
+
+engine:
+  # Set engine.config if you needa custom Dagger Engine config file
+  config: |
+    debug = true
+    insecure-entitlements = ["security.insecure"]
+    [registry."registry:5000"]
+      http = true
+    
+    [registry."docker.io"]
+      mirrors = ["mirror.gcr.io"]
+  resources:
+    ## Example:
+    ## limits:
+    ##    cpu: 100m
+    ##    memory: 128Mi
+    ##
+    limits: {}
+    ## Examples:
+    ## requests:
+    ##    cpu: 100m
+    ##    memory: 128Mi
+    ##
+    requests: {}
+  image:
+    repository: registry.dagger.io/engine
+    # tag: v0.6.5
```

Comment:
Should we link to something like https://github.com/dagger/dagger/blob/main/CHANGELOG.md which includes all available versions?